internal/rest/receipt/leveldb.go

```diff
@@ -44,7 +44,7 @@ type levelDBReceipts struct {
 func newLevelDBReceipts(conf *conf.ReceiptsDBConf) *levelDBReceipts {
 	store := kvstore.NewLDBKeyValueStore(conf.LevelDB.Path)
 	t := time.Unix(1000000, 0)
-	entropy := ulid.Monotonic(rand.New(rand.NewSource(t.UnixNano())), 0)
+	entropy := ulid.Monotonic(rand.New(rand.NewSource(t.UnixNano())), 0) // #nosec
```

Comment:
I'm curious why the `#nosec` on this one?

Reply:
It actually throws this error:
```
internal/rest/receipt/leveldb.go:47:28: G404: Use of weak random number generator (math/rand instead of crypto/rand) (gosec)
        entropy := ulid.Monotonic(rand.New(rand.NewSource(t.UnixNano())), 0)
```